scripts/extract/extractor/__init__.py

```diff
@@ -1,25 +1,27 @@
 import os
-module = os.environ.get('EXTRACTOR','NULL')
+module = os.environ.get('EXTRACTOR', 'NULL')
 
-if module=='onnx':
+if module == 'onnx':
     from .extract_onnx import OnnxExtractor as Extractor
-elif module=='caffemodel':
+elif module == 'caffemodel':
```

Comment:
```suggestion
elif module == 'caffe':
```

Reply:
EXTRACTOR not contains 'caffemodel', 'netdef', 'h5', 'mxnetparams ' ...

Reply:
This may be a problem, but it should‘t be solved this time . These modifications are just about code style.